acceptance/suites/tests/00_smoke/puppetdb_integration.rb

```diff
@@ -0,0 +1,52 @@
+## Tests that PuppetDB can be integrated with Puppet Server in a simple
+## monolithic install (i.e. PuppetDB on same node as Puppet Server).
+##
+## In this context 'integrated' just means that Puppet Server is able to
+## communicate over HTTP/S with PuppetDB to send it information, such as
+## agent run reports.
+##
+## This test validates communication is successful by querying the PuppetDB HTTP
+## API and asserting the agent's report timestamp is not null. This means that
+## PuppetDB successfully received the agent's report sent from Puppet Server.
+## We can just run the agent that's on the master for this.
+
+confine :to, {:platform => puppetdb_supported_platforms}, master
+
+require 'json'
+
+test_name 'PuppetDB integration'
+
+step 'Install PuppetDB module' do
+  on(master, puppet("module install puppetlabs-puppetdb --version 5.0.0"))
```

Comment:
I'd probably go with either unpinning it or pinning it with the option to configure it externally like we do with the repo configs.  I'm good however you want to go with this for now.